programs/mango-v4/tests/test_ix_gate_set.rs

```diff
@@ -34,22 +33,17 @@ async fn test_ix_gate_set() -> Result<(), TransportError> {
     .create(solana)
     .await;
 
-    let account = send_tx(
-        solana,
-        AccountCreateInstruction {
-            account_num: 0,
-            token_count: 8,
-            serum3_count: 7,
-            perp_count: 0,
-            perp_oo_count: 0,
-            group,
-            owner,
-            payer,
-        },
+    let account = create_funded_account(
+        &solana,
+        group,
+        owner,
+        0,
+        &context.users[1],
+        &mints[0..1],
+        10,
+        0,
     )
-    .await
-    .unwrap()
-    .account;
+    .await;
```

Comment:
you kept the explicit deposit that follows after this; probably that can be removed?